.github/workflows/linux.yml

```diff
@@ -976,29 +997,36 @@ jobs:
 
       - name: Extract OpenVINO packages
         run: |
-          pushd ${{ env.INSTALL_DIR }}
-            tar -xzf openvino_package.tar.gz -C ${{ env.INSTALL_DIR }} && rm openvino_package.tar.gz || exit 1
+          pushd ${INSTALL_DIR}
+            tar -xzf openvino_package.tar.gz -C ${INSTALL_DIR} && rm openvino_package.tar.gz || exit 1
           popd
-          pushd ${{ env.INSTALL_TEST_DIR }}
-            tar -xzf openvino_tests.tar.gz -C ${{ env.INSTALL_DIR }} && rm openvino_tests.tar.gz || exit 1
+          pushd ${INSTALL_TEST_DIR}
+            tar -xzf openvino_tests.tar.gz -C ${INSTALL_DIR} && rm openvino_tests.tar.gz || exit 1
           popd
-      - name: Install Python wheels
-        run: |
-          python3 -m pip install openvino --find-links=${{ env.INSTALL_DIR }}/tools
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+      - name: Setup pip cache dir
+        run: echo "PIP_CACHE_DIR=${PIP_CACHE_PATH}/$(python3 -m pip --version | grep -o 'pip [0-9]*[.][0-9]*[.][0-9]*' | grep -o '[0-9].*')" >> $GITHUB_ENV
+
+      - name: Install OpenVINO Python wheels
+        run: python3 -m pip install ${INSTALL_DIR}/tools/openvino-*
```

Comment:
Please revert this change. Only `openvino` is needed

Reply:
only `openvino` will be installed, because `openvino-dev` is named `openvino_dev-2023.2.0-12573-py3-none-any.whl`

Reply:
anyway, let us explicitly say only `openvino` to install. Just to avoid any other redundant deps.

Reply:
when we create release branch 2023.2 and new 2023.3 release is out after some time, this `pip install openvino` will install newer 2023.3 OpenVINO version from PyPi instead of locally built ones.

Current solution is to explicitly install local file to avoid such issues. We often see that older release branches are broken because of such excesses

Reply:
Got it, thanks